src/com/palmergames/bukkit/towny/command/TownCommand.java

```diff
@@ -64,6 +67,7 @@
 		output.add(ChatTools.formatCommand("", "/town", "list", ""));
 		output.add(ChatTools.formatCommand("", "/town", "online", TownySettings.getLangString("town_help_10")));
 		output.add(ChatTools.formatCommand("", "/town", "leave", ""));
+		output.add(ChatTools.formatCommand("", "/town", "evict", ""));

```

Comment:
The evict command just sends a player who isn't a resident of your town, out of the town somewhere?